packages/monaco/src/browser/monaco-editor-provider.ts

```diff
@@ -351,7 +351,7 @@ export class MonacoEditorProvider {
                 uri,
                 readContents: async () => '',
                 dispose: () => { }
-            }, this.m2p, this.p2m);
+            }, this.m2p, this.p2m, { encoding: this.editorPreferences.get('files.encoding') });
```

Comment:
inline editors never read anythign from the disk, they are for repls and brekapoints editing

Reply:
@akosyakov Sorry, I mistook the concept of 'inline editor'. I thought it has something to do with the editor shown in zone widget. I'm fixing it now. But what is the 'repls', by the way?